api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -0,0 +1,2168 @@
+            the value specified in SecurityContext takes precedence for that container.
+          title: Runasuser
+          type: integer
+        seLinuxOptions:
+          allOf:
+          - $ref: '#/components/schemas/SELinuxOptions'
+          description: The SELinux context to be applied to all containers. If unspecified,
+            the container runtime will allocate a random SELinux context for each
+            container.  May also be set in SecurityContext.  If set in both SecurityContext
+            and PodSecurityContext, the value specified in SecurityContext takes precedence
+            for that container.
+          title: Selinuxoptions
+        supplementalGroups:
+          description: A list of groups applied to the first process run in each container,
+            in addition to the container's primary GID.  If unspecified, no groups
+            will be added to any container.
+          items:
+            type: integer
+          title: Supplementalgroups
+          type: array
+        sysctls:
+          description: Sysctls hold a list of namespaced sysctls used for the pod.
+            Pods with unsupported sysctls (by the container runtime) might fail to
+            launch.
+          items:
+            $ref: '#/components/schemas/Sysctl'
+          title: Sysctls
+          type: array
+        windowsOptions:
+          allOf:
+          - $ref: '#/components/schemas/WindowsSecurityContextOptions'
+          description: Windows security options.
+          title: Windowsoptions
+      title: PodSecurityContext
+      type: object
+    PortworxVolumeSource:
+      properties:
+        fsType:
+          description: FSType represents the filesystem type to mount Must be a filesystem
+            type supported by the host operating system. Ex. "ext4", "xfs". Implicitly
+            inferred to be "ext4" if unspecified.
+          title: Fstype
+          type: string
+        readOnly:
+          description: Defaults to false (read/write). ReadOnly here will force the
+            ReadOnly setting in VolumeMounts.
+          title: Readonly
+          type: boolean
+        volumeID:
+          description: VolumeID uniquely identifies a Portworx volume
+          title: Volumeid
+          type: string
+      required:
+      - volumeID
+      title: PortworxVolumeSource
+      type: object
+    PreferredSchedulingTerm:
+      properties:
+        preference:
+          allOf:
+          - $ref: '#/components/schemas/NodeSelectorTerm'
+          description: A node selector term, associated with the corresponding weight.
+          title: Preference
+        weight:
+          description: Weight associated with matching the corresponding nodeSelectorTerm,
+            in the range 1-100.
+          title: Weight
+          type: integer
+      required:
+      - preference
+      - weight
+      title: PreferredSchedulingTerm
+      type: object
+    PriorityClass:
```

Comment:
This should be removed, I believe?

Reply:
I removed it to hide it from the API. Thanks!